client/src/components/pages/utils/renderButton.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const renderButton = (activeContent, text, showContent) => {
```

Comment:
do not extract render functions out into helper modules. Notice that the function is tightly coupled to the component - the `teach-profile__tap` classname, `showContent` function. Of course you could make it more generic, but then it's actually a **separate component** and should be placed among other components and have all the required ceremonies like propTypes, defaultProps etc.
It doesn't make sense to have this as a separate component since it only renders a `button` component. So pls move it back into TeacherProfile (but of course keep it as a separate function)